src/app/store/accounts/blockchain/bitcoin/taproot-account.hooks.ts

```diff
@@ -37,6 +37,10 @@ function useDeriveTaprootAccountIndexAddressIndexZero(xpub: string) {
   );
 }
 
+export function useKeychain() {
```

Comment:
This is used outside taproot specific file, so would name more specifically
```suggestion
export function useTaprootKeychain() {
```

Reply:
Thank you for fixing it in https://github.com/hirosystems/stacks-wallet-web/pull/3198/commits/1f35d4d48844b6e3f7e70671834de8fa01af7b73#diff-5e2bb9f387898c0f3a97e0ab57893591e28d9f12f23bae9c873f87c61543a11d